### PR TITLE
upgrade web3 package version

### PR DESCRIPTION
**Pull request by fofaraz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "ajv-errors": "^3.0.0",
     "axios": "^0.21.1",
     "bignumber.js": "^9.0.1",
```
Comment (bakhshandeh) [inline package.json:40]: Do we need both bignumber.js and bn.js?

File: src/utils/sha3.ts [hunk 2]
```diff
@@ -3,7 +3,10 @@ import jsSha3 from 'js-sha3'
 const web3Instance = new Web3()
 
 export function soliditySha3(params) {
-    return web3Instance.utils.soliditySha3(...params)
+  if (Array.isArray(params))
+    return web3Instance.utils.soliditySha3(...params);
+  else
```
Comment (bakhshandeh) [inline src/utils/sha3.ts:8]: Is this `else` necessary? :)

File: package.json [hunk 3]
```diff
@@ -63,14 +64,15 @@
     "multiformats": "^9.6.4",
     "node-cache": "^5.1.2",
     "nodemon": "^2.0.7",
+    "number-to-bn": "^1.7.0",
```
Comment (bakhshandeh) [inline package.json:67]: Can't we use BN?
Is installing a new module necessary?

**Comment by bakhshandeh** [comment]
I deployed this PR on a node and noticed that the `reqId` generated by the new soliditySha3 differs from the other nodes running the previous version.
There is an issue and need to be solved.

Here is the request:
http://139.59.101.131:8080/v1/?app=tss&method=test

**Comment by fofaraz** [comment]
> I deployed this PR on a node and noticed that the `reqId` generated by the new soliditySha3 differs from the other nodes running the previous version. There is an issue and need to be solved.
> 
> Here is the request: http://139.59.101.131:8080/v1/?app=tss&method=test

We have made a change in [src/utils/sha3.ts] to the `soliditySha3` method, which now handles strings correctly. In the older version of web3, you could use `sha3` on string values like this: `web3Instance.utils.soliditySha3(...params)`, where `params` is a string. However, in the new version, it throws an error, so we had to fix this. As a result, the `calculateRequestId` response now differs from older versions.

```
calculateRequestId(request, resultHash) {
    return soliditySha3([
      {type: "address", value: request.gwAddress},
      {type: "uint256", value: soliditySha3(request.data.uid)},
      {type: "uint32", value: request.data.timestamp},
      {type: "uint256", value: request.appId},
      {type: "string", value: soliditySha3(request.method)},
      {type: "uint256", value: resultHash},
    ]);
  }
  ```
This method uses `{type: "string", value: soliditySha3(request.method)}` where `request.method` is a string value.

**Comment by bakhshandeh** [comment]
> We have made a change in [src/utils/sha3.ts] to the `soliditySha3` method, which now handles strings correctly. 

That PR is already merged. sha3.ts is the latest version on all nodes.

**Pull request merged**